systemtest/pom.xml

```diff
@@ -234,6 +234,15 @@
             <groupId>io.fabric8</groupId>
             <artifactId>kubernetes-model-coordination</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.fabric8</groupId>
+            <artifactId>openshift-model-operatorhub</artifactId>
+            <version>${fabric8.openshift-client.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.fabric8</groupId>
+            <artifactId>openshift-client-api</artifactId>
+        </dependency>
```

Comment:
You should probably use the dependency management in the main `pom.xml` and not add them only here.